src/test_get_simulated_data.py

```diff
@@ -340,7 +340,7 @@ def test_null_sigma_m_makes_nan():
             seed=1,
             type_m="continuous",
             sigma_y=0.5,
-            sigma_m=0,
+            sigma_m=0.5,
```

Comment:
? I think the point of the test is the failure when sigma_m=0, so the value of sigma_m in the test should not be changed?

Reply:
you are right, I changed it back, my bad